### ISSUE TITLE
seperate swf for each page in pdf optional

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#63]
```
This functionality is not avaliable?

I need the ability to produce seperate swf for each page in pdf document. 

so that my custom viewer will load page on demand and not all the pages. 

i do not want to load the entire swf. 

Is this possible. i will be using command prompt for this. 

a simillar project is workng at www.pubmatch.com but they are having 
issues. 

thank you.
```

Original issue reported on code.google.com by `sssha...@gmail.com` on 6 May 2010 at 11:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 18 May 2010 at 3:36
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
What is the status of this feature.
```

Original comment by `anandamr...@gmail.com` on 14 Jan 2011 at 10:51

**Comment by GoogleCodeExporter** [comment]
```
I would you like to know the status of this issue. Will be in a next version?
```

Original comment by `cfsanto...@gmail.com` on 5 May 2011 at 1:18

**Comment by GoogleCodeExporter** [comment]
```
Hi, 
this will be possible in next version of FlexPaper (1.4.3)
```

Original comment by `erik.eng...@devaldi.com` on 5 May 2011 at 9:50

**Comment by GoogleCodeExporter** [comment]
```
implemented in 1.4.5 which is now available for download
```

Original comment by `erik.eng...@devaldi.com` on 23 May 2011 at 8:26
- Changed state: **Fixed**
